dev-docs/how-prebid-on-amp-works.md

```diff
@@ -13,7 +13,8 @@ nav_section: prebid-amp
 # How Prebid on AMP Works
 {: .no_toc}
 
-{% include amp-deprecation-warning.md %}
+{: .alert.alert-warning :}
+The information on this page is out of date.  It will be updated soon with the new "[AMP RTC][RTC-Overview] over [Prebid Server][PBS]" method which is planned for the [1.4 release](https://github.com/prebid/Prebid.js/releases/tag/1.4.0).
```

Comment:
release info can be removed

Reply:
Removed this warning and the 'Pre-release' text in the title in 3c75f61